File: AdSurgeDemo/AdSurgeDemo/SettingManager.m
//
//  SettingManager.m
//  AdSurgeDevDemo
//
//  Created by katie on 2025/5/22.
//

#import "SettingManager.h"

@implementation SettingManager

+ (instancetype)sharedManager {
    static SettingManager *manager;
    static dispatch_once_t onceToken;
    dispatch_once(&onceToken, ^{
        manager = [[SettingManager alloc] init];
    });
    return manager;
}

@end

File: AdSurgeDemo-Swift/AdSurgeDemo-Swift/SettingManager.swift
//
//  SettingManager.swift
//  AdSurgeDemo-Swift
//
//  Created by katie on 2025/6/6.
//

import Foundation

class SettingManager {
    static let shared = SettingManager()

    var isMuted: Bool = false
    
    private init() {}
}
